services/API-service/src/scripts/json/countries.json

```diff
@@ -440,7 +440,6 @@
         ],
         "eapLink": "https://510ibfsystem.blob.core.windows.net/about-trigger/MWI-flashfloods-about.pdf",
         "enableEarlyActions": false,
```

Comment:
_This comment is unrelated so feel free to ignore it for now._

I'm also wondering about the implication of this `enableEarlyActions` flag.

If the list of early actions is empty for this country-hazard then this can be inferred to be false instead of this explicit config.

Reply:
Valid point. I don't recall the exact reasoning for why it was put in like this at the time (and if the functionality still works now as intended then). I think we can leave this until we are touching the early actions in some way somewhere.